truss/test_data/truss_container_fs/app/common/errors.py

```diff
@@ -1,10 +1,125 @@
-class Error(Exception):
```

Comment:
I'll buy dinner for whoever removes this messy `truss_container_fs` directory.  The copy and pasted code here is a poor pattern

Reply:
Yeah. I think I might do it in a separate PR before merging this one. It's making the dev cycle unreasonably slow too

Reply:
https://github.com/basetenlabs/truss/pull/259